java/ql/lib/semmle/code/java/dataflow/internal/TaintTrackingUtil.qll

```diff
@@ -155,12 +156,19 @@ private module Cached {
     any(AdditionalTaintStep a).step(src, sink) and model = "AdditionalTaintStep"
   }
 
+  /**
+   * A sanitizer in all global taint flow configurations but not in local taint.
+   */
+  cached
+  abstract class DefaultTaintSanitizer extends DataFlow::Node { }
+
```

Comment:
TaintTrackingUtil doesn't need to reference any of ListOfConstantsSanitizer - it just needs to activate it. We usually use a private import inside a private module that's otherwise empty as the pattern for that to make it clear that it's a reverse import that can only activate class extensions and not bring anything in scope.
```suggestion
private module DefaultTaintSanitizerExtensions {
  private import semmle.code.java.security.ListOfConstantsSanitizer
}
```